lib/action_network_rest/people.rb

```diff
@@ -41,6 +41,12 @@ def find_by_email(email)
       end
     end
 
+    def update(person)
```

Comment:
It seems like potentially we should also support a hash and an ID as input to be more flexible re the sort of data we are accepting if the consuming code does.not have a person object handy.

Is it possible to send an update with a single field in a hash to not overwrite the entire person with new data for eg?